lib/get-service-auth-result.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+/**
+ *Create the Service State
+ @param {string} state
+ */
+class GetServiceAuthResult {
+	constructor(model) {
```

Comment:
1. Please, follow jsdoc pattern of ES2015 classes: https://jsdoc.app/howto-es2015-classes.html
```
/** Class representing the result of the get-service-auth-custom-state command */
class GetServiceAuthResult {
    /**
     * Creates a GetServiceAuthResult from model.
     * @param {object} model the model received from PKI Express
     */
    constructor(model) {
.....

```
2. Use getters and setters for the customState variable (you need to update customState to _customState). Doing that, it will be better for intellisense and programmer experience.
```
constructor(mode) {
    /** @private */
    this._customState = model['customState'];
}

/**
 * Gets the custom state value
 * @returns {string} the customState value
 */
 get customState() { return this._customState; }
```